src/style.css

```
@@ -1 +1,35 @@
-/* write your styles here */
+body {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+#header {
+  background-color: blue;
+}
+
+.important {
+  background-color: red;
+}
+
+.interesting {
+  background-color: green;
+}
+
+.surprising {
+  background-color: orange;
+}
+
+#footer {
+  background-color: black;
+}
+
+body > div {
```

```suggestion
div {
```
why do you need body there?

---

sorry, I first made changes just for between blocks )
I will correct now